src/sst/core/model/python2/pymodel.cc

```diff
@@ -29,6 +29,7 @@ REENABLE_WARNING
 #include "sst/core/sst_types.h"
```

Comment:
Convert this over to Python 3. Don't worry about Python 2.

Reply:
@jjwilke 
Ok, I will create a new branch dedicated to the PR of this issue that starts on the current develop of the main sst-core repository. And I will create only 1 to 3 commits to do the all the job. Indeed, intermediary results is just boring for the final review.

I you can just validate the work done on your review (about const and cast), that would be great.

Reply:
@jjwilke 
The last commit do the conversion to Python 3.

In addition, I added the possibility  to use the setStatistic method on a subComponent.
Now, in the python file you can do : 

```
for p in range(nports):
  portA = a.setSubComponent("port%d"%p, "ctest.ActivePort")
  link = sst.Link("a:%d-b:%d" % (p,p))
  link.connect((portA,"outport","10ns"),(portB,"outport","10ns"))
#  portA.enableStatistics(["observed_numbers"], {"type":"sst.HistogramStatistic"})
  statA = portA.setStatistic("observed_numbers")
  statA.addParams({"type":"sst.HistogramStatistic"})
```

I'm currently working on the final branch that will be used for the PR.